data_structures/linked_list/__init__.py

```diff
@@ -22,10 +31,33 @@ def remove(self):
             self.size -= 1
             return item
 
-    def is_empty(self):
+    def is_empty(self) -> bool:
         return self.head is None
 
-    def __len__(self):
+    def __str__(self) -> str:
+        """
+        >>> linked_list = LinkedList()
+        >>> linked_list.add(23)
+        >>> linked_list.add(14)
+        >>> linked_list.add(9)
+        >>> print(linked_list)
+        9 --> 14 --> 23
+        """
+        if not self.is_empty:
+            return "Linked List is empty."
```

Comment:
```suggestion
            return ""
```
This should be an empty string which suggests that the list is empty.

Reply:
That's great (Although it might not be a good idea for production code but this one is educational).